### ISSUE TITLE
Multi-input for Functionaries

### ISSUE THREAD

**Issue by c00k133** [Teknologforeningen/teknologr.io#82]
It would be great to have the possibility of selecting multiple members when inserting new functionaries, similarly to how it is done when selecting members for a certain group.

Pictures attached for some clarity:

_Screencap 1: when selecting a functionary_
![2019-11-22-230042_1365x685_scrot](https://user-images.githubusercontent.com/23499634/69460246-52886e00-0d7c-11ea-82de-fa9f2875a978.png)

_Screencap 2: when selecting members for a group_
![2019-11-22-230510_1365x686_scrot](https://user-images.githubusercontent.com/23499634/69460358-9c715400-0d7c-11ea-8a02-a20b5050ca23.png)

In other words, it would be great to be able to select multiple members as in a group when selecting functionaries, supported by a listing (see screencap `2` vs `1`).
